### ISSUE TITLE
R compliance checks

### ISSUE THREAD

**Issue by khufkens** [computationales/rsofun#99]
Things to fix before submission

```
Unknown, possibly misspelled, fields in DESCRIPTION:
  ‘Remotes’

Strong dependencies not in mainstream repositories:
  BayesianTools
* checking whether package ‘rsofun’ can be installed ... WARNING
Found the following significant warnings:
  wrappersc.c:12:6: warning: type of ‘pmodel_f_’ does not match original declaration [-Wlto-type-mismatch]
  wrappersc.c:122:6: warning: type of ‘lm3ppa_f_’ does not match original declaration [-Wlto-type-mismatch]
See ‘/home/khufkens/GECO/rsofun.Rcheck/00install.out’ for details.

* checking R code for possible problems ... NOTE
run_lm3ppa_f_bysite: no visible global function definition for
  ‘object.size’
runread_lm3ppa_f: no visible binding for global variable ‘data’
Undefined global functions or variables:
  data object.size
Consider adding
  importFrom("utils", "data", "object.size")
to your NAMESPACE file.

Status: 2 WARNINGs, 1 NOTE
```

**Comment by khufkens** [comment]
Notes removed for function references.

Compilation errors are inconsistent across platforms, needs checks.

**Comment by khufkens** [comment]
closed for now with #104
